scripts/portList.js

```diff
@@ -0,0 +1,34 @@
+/*portList.js this file controls population of the portfolio list as well as functionality of the portfolio item article preview*/
+/*resource templating logic*/
+
+var portListArray = [];
+var portArticleArray = [];
+
+/*constructor for PortListItem object*/
+function PortListItem (options){
```

Comment:
you can replace the code inside this function with:
  for (key in options) {
    this[key] = options[key];
  }
so that if you want to add another property to your object you only have to add the property and you don't have to add another line of code (code does it automatically)